### DISCUSSION TITLE
How to connect it at web?

### DISCUSSION THREAD

**Discussion by allran** [neutrinolabs/xrdp#2296] [Q&A]
Is it any way to use client at web? For example, use [mstsc.js](https://github.com/citronneur/mstsc.js), [node-rdpjs](https://github.com/citronneur/node-rdpjs), or other.  Thanks!
I use mstsc.js, and can't connect. When i use [Microsoft Remote Desktop], and can connect ubuntu with xrdp success.

**Comment by matt335672** [comment]
xrdp provides a standard RDP server. You can connect to it with any valid RDP client.

I can't tell you how to use any of the JavaScript alternatives you've linked to - you'll need to ask their own user communities.

If you just want to connect from a web client, I suggest you look at installing [Apache Guacamole](https://guacamole.apache.org/) on your server, and then just connecting over https. That is known to work well with xrdp.
